### DISCUSSION TITLE
Extended Props for Resources

### DISCUSSION THREAD

**Discussion by lukas-frey** [vkurko/calendar#310] [Ideas]
Would it be possible to also support "extendedProps" for resources, not only events? I need to pass additional data to the resource that then should be displayed using `resourceLabelContent`, but it's not possible currently.

**Comment by vkurko** [comment]
I think it's possible. I'll implement it in a future release.

**Comment by inu-web** [comment]
First: great work so far Vladimir! Thank you for sharing this great piece of software! I'm playing around with the options and the huge amount of functionality to find out if I can use it for our demands.

It would be great to have a possibility to pass more information to the resources to enable filtering/selecting the resources we want to show in the different resource and timeline views

so +1 from me for this idea/request!

regards Gerald

**Comment by vkurko** [comment]
`extendedProps` in resources should be available since v3.5.0. Please check.

**Reply by lukas-frey** [reply]
Awesome! Thanks a lot :)
